ansible/manage-node.yml

```diff
@@ -3,12 +3,19 @@
   vars:
     epoch_exec: "{{ ansible_env.HOME }}/node/bin"
   tasks:
-  - name: start
+  - name: Checking the epoch daemon state
+    shell: "{{ epoch_exec }}/epoch pid | cat"
```

Comment:
I prefer "epoch ping", looks more reliable to me

Reply:
Yes, I`m using epoch ping in the new one.
I`m currently working on other improvements as well.